src/main/java/com/mastery/java/task/dao/EmployeeDao.java

```diff
@@ -0,0 +1,24 @@
+package com.mastery.java.task.dao;
+
+import com.mastery.java.task.dto.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDate;
+
+@Repository
+public interface EmployeeDao extends JpaRepository<Employee, Integer> {
+
+    public Employee getEmployeeByEmployeeId(int employeeId);
```

Comment:
why do you need this method? I think you may use 	T getOne(ID id); method from JpaRepository instead this

Reply:
yes, the standard method is completely satisfied, i removed the unnecessary one